src/global_mapper_ros/global_mapper_ros.h

```diff
@@ -3,9 +3,11 @@
 
 #include <csignal>
 #include <memory>
+#include <string>
+#include <vector>
 
 #include "ros/ros.h"
-#include <tf2_ros/transform_listener.h>
+#include "tf2_ros/transform_listener.h"
```

Comment:
http://stackoverflow.com/questions/3162030/difference-between-angle-bracket-and-double-quotes-while-including-heade

This answer is a little better, but still leaves the terms "system" and "local" too loosely defined.